client/state/automated-transfer/selectors.js

```diff
@@ -68,3 +69,21 @@ export const isEligibleForAutomatedTransfer = compose(
 	getEligibilityStatus,
 	getEligibility
 );
+
+/**
+ * Determine if eligibility warnings or holds exist for
+ * a site other than a business plan hold.
+ *
+ * @param {Object} state global app state
+ * @param {number} siteId requested site for transfer info
+ * @returns {boolean} False if there is one and only one hold/warning referring to missing business plan
+ */
+export const hasEligibilityMessagesOtherThanBusiness = ( state, siteId ) => {
+	const data = getEligibility( state, siteId );
+	const holds = get( data, 'eligibilityHolds', [] );
+	const warnings = get( data, 'eligibilityWarnings', [] );
+
+	return warnings.length > 0 ||
+		holds.length > 1 ||
+		( holds.length > 0 && ! includes( holds, 'NO_BUSINESS_PLAN' ) );
```

Comment:
we wouldn't want to push this formatting out into **master**. For sequences of boolean conditions like this we would at least want proper indenting with surrounding parentheses…

```js
return (
	warnings.length > 0 ||
	holds.length > 1 ||
	( hold.length > 0 && ! includes( holds, 'NO_BUSINESS_PLAN' ) )
);
```

…or we might also use a helper like `Array.some()`

```js
return some( [
	warnings.length > 0,
	holds.length > 1,
	holds.length > 0 && ! includes( holds, 'NO_BUSINESS_PLAN' ),
] );
```